packages/base/src/Dialog/Dialog.tsx

```diff
@@ -30,7 +30,7 @@ export const DialogBase: RneFunctionComponent<DialogBaseProps> = ({
       testID="Internal__Overlay"
       {...rest}
     >
-      <View style={styles.childrenContainer}>{children}</View>
```

Comment:
What are you trying to do here?

Reply:
as mentioned in the issue childrenContainer contains a margin-bottom which is not required so I just removed it.

Reply:
hey @arpitBhalla do you except something else from this PR, please let me know

Reply:
Maybe you can add a prop for containerStyle

Reply:
If you don't add a prop for the container style the view isn't even needed. Personally, I feel like the fewer props the better, which gives us more control of the children.

Reply:
Yeah we can do this, cause this is only a wrapper, @hitesh-coder can you replace View with children only